src/clients/host-client.js

```diff
@@ -233,6 +233,16 @@ class HostClient extends BaseEvernodeClient {
         return await this.isRegistered();
     }
 
+    async updateRegInfo(newRegInfo, options = {}) {
```

Comment:
In the 'register' function we have all inline parameters: `async register(countryCode, cpuMicroSec, ramMb, diskMb, totalInstanceCount, description, options = {})`
But in updateRegInfo() we have a 'newRegInfo' object. Why are they inconsistent?